backend/src/controllers/skills/getAllSkills.ts

```diff
@@ -0,0 +1,20 @@
+import { NextFunction, Request, Response } from 'express';
+
+import { getAllGroupsWithSkills, getAllSkillsByGroup } from 'db/providers/skillProvider';
+
+const getAllSkills = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    const searchStr = req.query;
+    const skillsGroups = await getAllGroupsWithSkills(searchStr);
+    if (skillsGroups.length) {
+      res.json(skillsGroups);
+      return;

```

Comment:
no empty returns please 🙏🏻

Reply:
sorry, but i don't agree with you, this return prevent  from execution the code below if statement( we don't have else statement), otherwise our server try to push again new headers to client